.github/workflows/python-poetry-release.yaml

```diff
@@ -89,8 +88,7 @@ jobs:
           poetry-version: ${{ inputs.poetry-version }}
 
       - name: Commit and push pyproject.toml file
-        if: success()
-        uses: bakdata/ci-templates/actions/commit-and-push@v1.0.0
+        uses: bakdata/ci-templates/actions/commit-and-push@v1.13.1
```

Comment:
afaict this action wasn't changed?

Reply:
No, it wasn't but I wanted to update it to the newest release (that we do after this merge).

Reply:
the update is technically unnecessary 🤷🏻